app/models/story.rb

```diff
@@ -64,7 +65,7 @@ def self.paranoia_scope
   # indicates the piece is not publically available, only to the network
   event_attribute :network_only_at
 
-  scope :published, -> { where('`published_at` IS NOT NULL AND `published_at` <= now()') }
+  scope :published, -> { where('`published_at` <= now()') }
```

Comment:
in mysql, if you are doing a comparison against a value, such as `<=`, then it will not match any records with a null value - not for `>` or `<`. So it is redundant in this case to exclude where it `is not null`, because by comparing against `now`, it already is.